### PR TITLE
Add checking for client_id and client_secret

### PR DESCRIPTION
**Pull request by LeifAndersen**

### CODE REVIEW

**Comment by jeapostrophe** [comment]
Drop the change to run.sh

**Comment by LeifAndersen** [comment]
Oh, woops, I didn't realize that got in there...thanks.

**Comment by LeifAndersen** [comment]
Fixed.

File: official/common.rkt [hunk 1]
```diff
@@ -25,8 +25,12 @@
 (define users.new-path (build-path root "users.new"))
 (make-directory* users.new-path)
 
-(github-client_id (file->string (build-path root "client_id")))
-(github-client_secret (file->string (build-path root "client_secret")))
+(if (file-exists? (build-path root "client_id"))
+    (github-client_id (file->string (build-path root "client_id")))
+    (error "Cannot find file ~a" (build-path root "client_id")))
```
Comment (jeapostrophe) [inline official/common.rkt]: Can you make this a valid call to error? Put the symbol as 'pkg-index

File: official/common.rkt [hunk 2]
```diff
@@ -25,8 +25,12 @@
...
+(if (file-exists? (build-path root "client_scret"))
+    (github-client_secret (file->string (build-path root "client_secret")))
+    (error "Cannot find file ~a" (build-path root "client_secret")))
```
Comment (jeapostrophe) [inline official/common.rkt]: Ditto

**Comment by LeifAndersen** [comment]
Okay, I changed them to be user errors.

**Pull request merged**